src/mbedtls/pk.pyx

```diff
@@ -81,9 +81,10 @@ class Curve(bytes, enum.Enum):
     BRAINPOOLP384R1 = b'brainpoolP384r1'
     BRAINPOOLP512R1 = b'brainpoolP512r1'
     CURVE25519 = b'curve25519'
-    SECP256K1 = b'secp256k1'
-    SECP224K1 = b'secp224k1'
     SECP192K1 = b'secp192k1'
+    SECP224K1 = b'secp224k1'
+    SECP256K1 = b'secp256k1'
+    CURVE448 = b'curve448'
```

Comment:
I would rather group the ```CURVE*``` names so that they are easier to find later on.

Reply:
This list is merely `enum mbedtls_ecp_group_id` from `mbedtls/ecp.h`, more or less "sorted by enum value". I think any order should be the same in python enum. You could reorder them anyway convenient.